### PR TITLE
pythonPackages.python-ldap-test: init at 0.3.1

### PR DESCRIPTION
**Pull request by PsyanticY**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/python-ldap-test/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qh9x3lncaldnw79fgpqbayichs8pbz8abr6pxb5qxbs7zrnyrwf";
+  };
+
+  propagatedBuildInputs = [ py4j ];
+
+  # Tests needs java to be present in path
+  doCheck = false;
```
Comment (Ma27) [inline pkgs/development/python-modules/python-ldap-test/default.nix]: would it be sufficient to add Java as checkInput and alter `$PATH` in the `checkPhase`?

I'm not entirely sure yet if we actually want to add `java` to the closure.

Comment (Assassinkin) [reply pkgs/development/python-modules/python-ldap-test/default.nix]: @Ma27  I think that might work. Nonetheless, I checked the other python packages and I couldn't find any one using JAVA so i dropped the idea.

Comment (Ma27) [reply pkgs/development/python-modules/python-ldap-test/default.nix]: true, but I don't know if there are that much other python packages that require java for any part of the build process :)

Comment (FRidh) [reply pkgs/development/python-modules/python-ldap-test/default.nix]: Aside from using unit tests, I doubt we'll be able to actually test speaking to an LDAP server during a build :)

Comment (Assassinkin) [reply pkgs/development/python-modules/python-ldap-test/default.nix]: So, Should we enable the tests or not?  :confused:

**Review by Ma27 — APPROVED** [review]
just built python {2,3} versions locally and played around with it :+1:

**Pull request merged**